Source/Scene/ModelInstanceCollection.js

```diff
@@ -131,6 +132,7 @@ define([
         this._gltf = options.gltf;
         this._basePath = options.basePath;
         this._asynchronous = options.asynchronous;
+        this._shadows = option.shadows;
```

Comment:
It would also be nice to make `shadows` settable whenever. Add a getter/setter `shadows` property to the `defineProperties` section that sets `_model.shadows`.